app/views/schemes/changes.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <h1 class="govuk-heading-l">Schemes and locations</h1>
+    <h2 class="govuk-heading-m">What is a scheme?</h2>
+    <p class="govuk-body">A supported housing scheme (also known as a ‘supported housing service’) provides shared or self-contained housing for a particular client group, for example younger or vulnerable people.</p>
+
+    <h2 class="govuk-heading-m">What is a location?</h2>
+    <p class="govuk-body">A location is a postcode where supported housing is provided under a scheme. A scheme can have multiple locations, and a location can have multiple units at the same postcode.</p>
+
+    <h2 class="govuk-heading-m">How schemes have changed</h2>
+    <p class="govuk-body">We have restructured the way we group supported housing scheme data.</p>
+    <p class="govuk-body">On old CORE, a scheme’s data was split into management group and scheme. On new CORE, the data is split into scheme and location.</p>
+    <p class="govuk-body">These are the main changes:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Schemes now store data about the stock owner and support provider. This was previously stored in management group.</li>
+      <li>Schemes still store data about the type of support, as they did in old CORE.</li>
+      <li>Schemes no longer store data about postcodes where support is provided. These are now in locations.</li>
+    </ul>
+    <p class="govuk-body">This new structure means data coordinators only needs to fill in support details once.</p>
+    <h2 class="govuk-heading-m">How schemes are migrated from old CORE</h2>
+    <p class="govuk-body">If your organisation has migrated from old CORE to new CORE, your existing schemes are on the <%= govuk_link_to("schemes page", schemes_path) %>, with the same details.</p>
```

Comment:
is old CORE and new CORE the correct terminology or just something we've been using internally? Seems a bit casual, I'd have expected something closer to "the previous version of this service" but arguably that's less clear